cmd/images-back/main.go

```diff
@@ -10,29 +10,66 @@ import (
 	"os/signal"
 	"strconv"
 	"time"
+	_ "github.com/lib/pq"
+	"database/sql"
 )
 
 var port int
+var db string
 
 func init() {
-	defPort := 5000
+	defPort := 8080
+	var defDB string
 	if portVar, ok := os.LookupEnv("PORT"); ok {
 		if portValue, err := strconv.Atoi(portVar); err == nil {
 			defPort = portValue
 		}
 	}
+	if dbVar, ok := os.LookupEnv("DATABASE_URL"); ok {
+		defDB = dbVar
+	}
 	flag.IntVar(&port, "port", defPort, "port to listen on")
+	flag.StringVar(&db, "db", defDB, "database to connect to")
 }
 
 func main() {
 	flag.Parse()
 	e := echo.New()
-	e.Static("/static", "static")
 	e.Logger.SetLevel(log.INFO)
+
 	e.GET("/", func(c echo.Context) error {
 		return c.String(http.StatusOK, "Hello, World!")
 	})
-	e.File("/index", "static/index.html")
+
+	e.GET("dbTest", func(c echo.Context) error {
+		//connStr := "user=images password=images_go dbname=imagesapp sslmode=disable"
+		url := db
+		dbPsql, err := sql.Open("postgres", url)
```

Comment:
The `sql.Open()` is heave-weight operation. It is better to open a connection to the database once on startup and use it everywhere, instead of opening and closing it in handlers.

Go driver has a connection pool under the hood, so it is ok to use one connection for all handlers.